packages/glibc.rb

```diff
@@ -3,29 +3,33 @@
 class Glibc < Package
   description 'The GNU C Library project provides the core libraries for GNU/Linux systems.'
   homepage 'https://www.gnu.org/software/libc/'
-  version '2.23'
-  source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.23.tar.xz'
-  source_sha256 '94efeb00e4603c8546209cefb3e1a50a5315c86fa9b078b6fad758e187ce13e9'
+  version '2.27'
+  source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.27.tar.xz'
```

Comment:
This really should be `https://ftpmirror.gnu.org/libc/glibc-2.27.tar.xz`. `glibc` is a symlink to `libc`.

Reply:
What exactly should be changed here?